js/package.json

```diff
@@ -2,12 +2,17 @@
   "private": true,
   "name": "@flarum/mentions",
   "dependencies": {
-    "flarum-webpack-config": "0.1.0-beta.10",
-    "webpack": "^4.43.0",
-    "webpack-cli": "^3.3.11"
+    "flarum-webpack-config": "0.1.0-beta.16.2",
+    "webpack": "^4.46.0",
+    "webpack-cli": "^4.7.0"
   },
   "scripts": {
     "dev": "webpack --mode development --watch",
     "build": "webpack --mode production"
+  },
+  "devDependencies": {
+    "@types/mithril": "^2.0.7",
+    "flarum": "^0.1.0-beta.16",
+    "flarum-tsconfig": "^0.1.0-beta.16.1"
```

Comment:
are these packages still on a beta versioning btw ?

Reply:
`flarum` was used for typings, can be removed as we use `vendor` folder now - tsconfig is on v1.0.0